symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -128,6 +93,13 @@ public ActivityRegistry activities() {
     return this.activityRegistry;
   }
 
+  public AuthSession obo(Obo.Handle oboHandle) throws AuthUnauthorizedException {
```

Comment:
In the end, do we still really need this `Obo.Handle`? Maybe just 2 methods : `obo(String)` and `obo(Long)` can do the job now?